kubernetes/calico-node.env

```diff
@@ -0,0 +1,6 @@
+ETCD_AUTHORITY=127.0.0.1:4001
+HOSTNAME=dev
+IP="10.0.2.15"
```

Comment:
This doesn't exist for me.  Should this be the docker host?

IP=$(echo $DOCKER_HOST | awk -F'[/:]' '{print $4}')